tests/components/dlna_dmr/test_media_player.py

```diff
@@ -1274,6 +1274,7 @@ async def test_unavailable_device(
     hass.config_entries.async_update_entry(
         config_entry_mock, options={CONF_POLL_AVAILABILITY: True}
     )
+    await hass.async_block_till_done()
```

Comment:
Since there is no longer a task being created in `async_update_entity` the `async_update_entry` call wasn't finishing before the entity update started.